lib/ruby_lsp/event_emitter.rb

```diff
@@ -251,5 +251,101 @@ def visit_block_local_variable_node(node)
       @listeners[:on_block_local_variable]&.each { |l| T.unsafe(l).on_block_local_variable(node) }
       super
     end
+
```

Comment:
Not for this PR, but as followup maybe we can order these alphabetically?

Reply:
We're doing work to ship the event emitter in YARP, so maybe let's hold off since we may be able to get rid of this entire file.